frontend/src/routes/(authenticated)/project/[project_code]/+page.svelte

```diff
@@ -39,6 +39,11 @@
   let projectStore = data.project;
   $: project = $projectStore;
   $: _project = project as NonNullable<typeof project>;
+  $: changesetPromise = data.promise.changesets.then((result) => {
+    return new Promise((fulfill) => {
+      result.projectByCode.subscribe((x) => { if (x) { fulfill(x.changesets); } });
```

Comment:
I think we theoretically need to unsubscribe again?
Other than that this all looks good to me and it's a great improvement!

We are losing the "query-store" functionality, which currently doesn't matter, but...

I think a big part of the trickiness here is that `queryStore` returns a promise with stores in it instead of just a store. @hahn-kev it's not obvious to me why that's the case. Could `queryStore` return a store?

Reply:
it could but then you would have no easy way of awaiting when the loading was done. But to that end I don't think we benefit from using the query store here. We can just use query on the gql client. That just returns a promise and then we don't need to fuss with the store here, the reason we use the store is so that mutations can return updated results and they make it back to the page, but I don't see us needing that for changesets so we can skip it here.

Reply:
Ah gotcha. Right, you'd lose the ability to await results and more code would potentially have to be null safe.

But yes, our mutations will presumably never affect change sets, so we don't need a query-store here, in which case @rmunn you could juse use `client.query` instead and you'd probably be able to go back to the tidy `{#await}` syntax.

Reply:
Actually, for the project-reset we could use a query-store here. I added an [explicit call to `invalidate()`](https://github.com/sillsdev/languageforge-lexbox/pull/335/files#diff-e97283ee35e29a60f69f82263486f1dc5de784040ba9d2ae3e495cfbf1924223R208), anticipating this not being a query-store anymore, but if this were to remain a functioning query-store, we wouldn't need that.